spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-ai/src/test/java/com/alibaba/cloud/ai/tongyi/chat/TongYiChatOptionsTests.java

```diff
@@ -37,14 +37,14 @@ class TongYiChatOptionsTests {
 	public void testChatOptions() {
 
 		Generation mockClient = Mockito.mock(Generation.class);
-		Constants.apiKey = "test";
+		Constants.apiKey = "sk-xxx";
```

Comment:
pls restore this changes, this change is not directly related to this pr.

we should maintain the singularity of pr